server/chat-commands/admin.ts

```diff
@@ -210,6 +210,25 @@ export const commands: ChatCommands = {
 		`/pmuhtmlchange [user], [name], [html] - Changes html that was previously PMed to [user] to [html]. Requires * # &`,
 	],
 
+	sendhtmlpage(target, room, user, connection) {
+		if (!this.can('addhtml', null, room)) return false;
+		let [targetID, pageid, content] = Utils.splitFirst(target, ',', 2);
+		if (!target || !pageid || !content) return this.parse(`/help sendhtmlpage`);
+		const targetUser = Users.get(targetID);
+		if (!targetUser) return this.errorReply(`User not found.`);
+		content = this.canHTML(content);
```

Comment:
```suggestion
		content = this.canHTML(content)!;
```

whatever, we can deal with this in other ways.